test/data/test_utils.py

```diff
@@ -45,3 +46,19 @@ def test_get_tokenizer_toktokt(self):
             data.get_tokenizer(1)
         with self.assertRaises(ValueError):
             data.get_tokenizer("some other string")
+
+    def test_text_nomalize_function(self):
+        # Test text_nomalize function in torchtext.datasets.text_classification
+        ref_lines = []
+        test_lines = []
+        with open('../asset/test.csv') as src_data:
```

Comment:
can you name this something more expressive? Maybe "text_normalization_ag_news_test.csv" or such. Or move it into a different folder "reference" and the .test dataset into "data".

Reply:
let me rename it.